### PR TITLE
SM: add functionality to remove and update items

### PR DESCRIPTION
**Pull request by smathis122**
Remove and Update items completed

### CODE REVIEW

File: code/Frontend/react-frontend/src/pages/AddItem.js [hunk 1]
```diff
@@ -12,36 +12,61 @@ function EquipmentForm() {
...
+    // Get the user ID obtained during login
+    const userId = getUserId(); // Implement a function to retrieve the user ID
+
+    // Create the equipment object with the user's ID as the owner
+    const newEquipment = {
+      name: formData.name,
+      description: formData.description,
+      status: formData.status,
+      price: formData.price,
+      owner: userId, // Include the user's ID as the owner
```
Comment (casejade37) [inline code/Frontend/react-frontend/src/pages/AddItem.js:38]: I left out userId and ItemId in my Reservation part for this week due to time constraints - but I may borrow your pattern here to add those next week.

File: code/Frontend/react-frontend/src/pages/AddItem.js [hunk 2]
```diff
@@ -12,36 +12,61 @@ function EquipmentForm() {
...
+    };
 
-  const handleSubmit = (event) => {
-    event.preventDefault();
-    // Send the formData as JSON to your Flask back-end here
-    fetch("http://127.0.0.1:5000/api/addEquipment", {
+    // Send a POST request to your API to create the equipment
+    fetch("http://your-api-url/api/addEquipment", {
```
Comment (casejade37) [inline code/Frontend/react-frontend/src/pages/AddItem.js:42]: How does this work? I assume your-api-url is a variable created somewhere but I can't find it.

**Review by casejade37 — APPROVED** [review]
Great as always, textbook style of codes for the rest of the team to follow

File: code/Frontend/react-frontend/src/pages/AddItem.js (hunk 2 above)
Comment (smathis122) [reply code/Frontend/react-frontend/src/pages/AddItem.js:42]: Yeah this was a mistake - it shouldn’t work - I will fix it

**Review by ahnaft7 — APPROVED** [review]

**Pull request merged**